code/game/gamemodes/cult/cult_structures.dm

```diff
@@ -135,7 +133,7 @@
 			head.disfigure("burn") //Your face is unrecognizable because it's FUCKING LAVA
 		return 1
 
-/obj/structure/cult/cultforge/attack_hand(mob/living/user)
+/obj/structure/cult/itemspawner/forge/attack_hand(mob/living/user)
```

Comment:
`attack_hand` can be all defined on the `/obj/structure/cult/itemspawner` level, removing a bunch of duplicated code